app/index.jsx

```diff
@@ -2,8 +2,23 @@ var React = require('react')
 var ReactDOM = require('react-dom')
 const constants = require('./scripts/constants.js')
 
-var Nav = require('./components/nav.jsx')
-var BootstrapPre = require('./components/bootstrap-pre.jsx')
+var Navigation = require('./components/navigation.jsx')
+const componentCode = [
```

Comment:
Don't require the code into an array.

Couple things to do here:

Create a folder called app/components/elements
Move the elements you've created into the above folder (alerts-pre, button-groups-pre) etc
Create a file called app/components/elements.jsx
In elements.jsx, do something like:

```
module.exports = {
  Alert: require('./components/elements/alerts-pre.jsx),
  ButtonDropdown: require('./components/elements/button-dropdowns-pre.jsx),
  etc...
}

Then in index.jsx, you can do `var elements = require('./elements.jsx')`
From then on, you can then refer to each element by key. And if you really need to at any point, loop through the elements by key and value.